etc/kayobe/inventory/group_vars/mgmt-switches

```diff
@@ -0,0 +1,122 @@
+---
+# Management switch configuration.
+
+###############################################################################
+# Switch credentials.
+
+# User to access the switch via SSH.
+ansible_user: "{{ secrets_switch_ssh_username }}"
+
+# Using SSH public key authentication.
+#ansible_ssh_pass: "{{ secrets_switch_ssh_password }}"
+
+# Password authentication is after login.
+switch_auth_pass: "{{ secrets_switch_ssh_password }}"
+
+###############################################################################
+# Global configuration.
+
+# Global configuration. List of global configuration lines.
+switch_config_all:
+  - "hostname \"{{ inventory_hostname }}\""
+  - vlan database
+  - "vlan {{ switch_config_vlans | join(',') }}"
+  - exit
+
+# List of VLANs configured on these switches.
+switch_config_vlans:
+  - "{{ ilab_vlan }}"
+  - "{{ devops_vlan }}"
+  - "{{ oc_oob_vlan }}"
+  - "{{ oc_provision_vlan }}"
+  - "{{ wl_oob_vlan }}"
+  - "{{ wl_provision_vlan }}"
+  - "{{ wl_cleaning_vlan }}"
+  - "{{ wl_inspection_vlan }}"
+  - "{{ internal_vlan }}"
+  - "{{ storage_vlan }}"
+  - "{{ storagemgmt_vlan }}"
+  - "{{ switch_interface_config_tenant_vlan_range }}"
+
+###############################################################################
+# Interface configuration.
+
+# Interface config for overcloud iDRACs.
+switch_interface_config_oc_idrac:
+  - spanning-tree portfast
```

Comment:
Does Cambridge use spanning tree on this network?  I am not sure that they do.  It might be better to disable it globally if that is the site policy.

Reply:
This line of configuration was applied to all interfaces when I started working on the switch. Not sure if it was put there intentionally or not.